README.rst

```diff
@@ -1,95 +1,167 @@
-=========
+#########
 I Promise
-=========
+#########
+
 .. image:: https://badge.fury.io/py/ipromise.svg
     :target: https://badge.fury.io/py/ipromise
 
-This repository provides a Python base class, and various decorators for specifying promises relating to inheritance.
-It provides three inheritance patterns:
+This repository, ``ipromise``, provides a Python base class and
+decorators to guarantee inheritance "promises".
+
+Three inheritance patterns are provided:
+
+* implementing
+* overriding
+* augmenting
+
+Using the ``ipromise`` inheritance patterns can ensure an inheritance hierarchy
+is used as intended by forcing a build-time or run-time failure when not.
+
+----
+
+.. contents::
+
+----
+
+**********
+Installing
+**********
 
-* implementing,
-* overriding, and
-* augmenting.
+.. code-block:: shell
 
-Base class
-==========
-Checking promises depends on inheritance from the base class ``AbstractBaseClass``.  Unlike the standard library's similar class ``abc.ABCMeta``, ``AbstractBaseClass`` does not bring in any metaclasses.  This is thanks to Python 3.6's PEP 487, which added ``__init_subclass__``.
+    python -m pip install ipromise
+
+***
+Use
```

Comment:
Does this section add anything?

Reply:
I didn't see the table of contents, but okay let's just keep it then.  I'll remove it finally if it doesn't look good.

Reply:
It's the statement `.. contents::`

https://docutils.sourceforge.io/docs/ref/rst/directives.html#table-of-contents

I added VS Code extension _reStructuredText_ to view the README.rst.

Reply:
Awesome, thanks.